Rakefile.rb

```diff
@@ -2,7 +2,21 @@
 require 'rubocop/rake_task'
 
 Rake::TestTask.new do |t|
-  t.pattern = 't/**/*.rb'
+  t.warning = true
+  t.test_files = FileList['t/page/*.rb']
+end
```

Comment:
It might be worth adding the (undocumented, as far as I can tell) `t.description = 'Blah'` option here.

That would make the output of `rake -T` a bit more useful:

```
$ be rake -T
rake rubocop               # Run RuboCop
rake rubocop:auto_correct  # Auto-correct RuboCop offenses
rake test                  # Run tests
rake test:all              # Run tests
rake test:web              # Run tests
```